vulture_os/toolkit/api_parser/cybereason/cybereason.py

```diff
@@ -116,15 +105,23 @@ def _connect(self):
         except Exception as err:
             raise CybereasonAPIError(err)
 
-    def execute_query(self, method, url, query, timeout=10):
+    def execute_query(self, method, url, query=None, header=HEADERS, data=None, timeout=10):
```

Comment:
timeout is a variable to configure time between tries, and not request timeout ? misleading name